.github/workflows/lint.yml

```diff
@@ -18,8 +18,8 @@ jobs:
           node-version: lts/*
 
       - run: |
-          npm install -g yarn
-          yarn install --frozen-lockfile
+          npm install -g pnpm
+          pnpm install --frozen-lockfile --shamefully-hoist=true
```

Comment:
We may be able to remove both options - see comment above

This also applies to all the other workflows that call `pnpm install` 😋